src/PromiseType.cc

```diff
@@ -133,7 +127,7 @@ JSObject *PromiseType::toJsPromise(JSContext *cx) {
   PyObject *futureCallbackTuple = PyTuple_Pack(2, PyLong_FromVoidPtr(cx), PyLong_FromVoidPtr(rootedPtr));
   PyObject *onDoneCb = PyCFunction_New(&futureCallbackDef, futureCallbackTuple);
   future.addDoneCallback(onDoneCb);
-
+  Py_INCREF(pyObject);
```

Comment:
I don't think this `Py_INCREF` is necessary, `loop.ensureFuture` should be managing this
```suggestion
```

Reply:
we crash without it